ENVIRONMENT.md

```diff
@@ -13,10 +13,17 @@ documented in the proper man pages.
 
 All tools:
 
-* `$SYSTEMD_IGNORE_CHROOT=1` — if set, don't check whether being invoked in a
-  chroot() environment. This is particularly relevant for systemctl, as it will
-  not alter its behaviour for chroot() environments if set. (Normally it
-  refrains from talking to PID 1 in such a case.)
+* `$SYSTEMD_OFFLINE=[0|1]` — This supercedes the `$SYSTEMD_IGNORE_CHROOT`
+  environment variable. If set to `1`, then tools such as `systemctl` will
+  refrain from talking to PID 1. Setting this variable to `0` will have the
+  same effect as `SYSTEMD_IGNORE_CHROOT=1`; i.e. tools will try to
+  communicate with PID 1 even if a `chroot()` environment is detected.
```

Comment:
hmm, i don't think we should claim it really supercedes it. After all SYSTEMD_IGNORE_CHROOT applies to all invocations of running_in_chroot() and there's more than just systemctl using it to enable offline mode. i.e. systemd itself uses it to make sure it's not run in a chroot env and give a friendly warning. Or the "systemd-detect-virt" tool honours it too.

So yes, it's not orthogonal, but it's also not a superset of each other, and we shouldn't claim so, or that one supercedes the other.

Hence, I think a slightly weaker language would be good, i.e. something saying that "primarily, SYSTEMD_IGNORE can be used to enforce offline mode, but actually, if that#s what you want to do use SYSTEMD_OFFLINE instead, because it's more explicit and specific"